.gitignore

```diff
@@ -16,3 +16,4 @@ VPinballX_*-alp4k
 **/._*
 !/external/*/pinmame/roms/.gitkeep
 
+external/vpx-nogoodgofers/pinmame/cfg/default.cfg
```

Comment:
what's this for? This won't prevent users from adding files, it only prevents them from being committed by git.

Reply:
Understood, I just didn't want to submit anything that wasnt necessary. To the repo